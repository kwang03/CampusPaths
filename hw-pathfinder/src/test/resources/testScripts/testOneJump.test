#Tests Dijkstra's when the least cost path has one edge

CreateGraph myGraph

AddNode myGraph A
AddNode myGraph B
AddNode myGraph C

AddEdge myGraph A B 1.0
AddEdge myGraph A C 1.0
AddEdge myGraph C B 1.0

FindPath myGraph A B