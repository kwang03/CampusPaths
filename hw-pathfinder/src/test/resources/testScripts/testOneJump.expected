#Tests Dijkstra's when the least cost path has one edge

created graph myGraph

added node A to myGraph
added node B to myGraph
added node C to myGraph

added edge 1.000 from A to B in myGraph
added edge 1.000 from A to C in myGraph
added edge 1.000 from C to B in myGraph

path from A to B:
A to B with weight 1.000
total cost: 1.000